dolphinscheduler-task-plugin/dolphinscheduler-task-sql/src/main/java/org/apache/dolphinscheduler/plugin/task/sql/SqlTask.java

```diff
@@ -302,6 +302,7 @@ private String executeQuery(Connection connection, SqlBinds sqlBinds, String han
     private String executeUpdate(Connection connection, List<SqlBinds> statementsBinds, String handlerType) throws Exception {
         int result = 0;
         for (SqlBinds sqlBind : statementsBinds) {
+            logger.info("{} statement execute update query, for sql: {}", handlerType, sqlBind.getSql());
```

Comment:
I don't understand why you did this...

Reply:
> I don't understand why you did this...

This log outputs the current sql statements while executing multiple sql statements so that user can understand which sql statement is executing. This is helpful when something goes wrong while executing multiple sql statements.

The original log can only output the current sql statement if the sql type is `Mysql`, but cannot output the current sql statement if the sql type is `Hive`.

Reply:
Hi, @SbloodyS , please review this.

I have added a log output in prepareStatementAndBind, outputting SQL and parameters.